test/coffeescript.coffee

```diff
@@ -30,7 +30,7 @@ describe 'coffeescript', ->
 				throw new MyError()
 			catch e
 
-			expect(e).to.have.nested.property('constructor.name', 'MyError')
+			expect(e).to.have.property('constructor').that.has.property('name', 'MyError');
```

Comment:
I have no clue why, but it seems that even in node 12 the `nested.property` check wasn't wokring :thinking: